ecosystem_cicd_tools/release.py

```diff
@@ -163,17 +163,14 @@ def plugin_release(plugin_name,
                    version=None,
                    plugin_release_name=None,
                    plugins=None):
-
     # plugins = plugins or {}
     version = version or get_plugin_version()
     plugin_release_name = plugin_release_name or "{0}-v{1}".format(
         plugin_name, version)
     version_release = get_release(version)
-    commit = get_commit()
     if not version_release:
         version_release = create_release(
-            version, version, plugin_release_name,
-            commit)
+            version, version, plugin_release_name)
```

Comment:
Why are you removing `commit`?

Reply:
because the commit is used for "target_commitish" and in our case it will be always be "master" because we  create new version from the master code.

Reply:
nopers! it will not always be master! In fact, it might never be master depending on if we change the release process!